usagov/usagov-2021
# USAGOV-1812-wizard-checked-property-vs-class: Rely on the wizard <input> element checked property rather than the checked class

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1812

## Description

Update wizard.scss to look for checked property instead of class, and delete wizardChecked.js.

## Type of Changes

- [ X ] Frontend (Twig, Sass, JS)
![image](https://github.com/user-attachments/assets/b3ea9965-fda5-41d6-9677-9479d6000133)

## Testing Instructions


1. Navigate to any wizard page other than the one with the question "Where did the scam take place?" that has an options list.
2. Check one of the options.
3. Open devtools and inspect the <label> element of the option clicked in step 2.
4. The element does not have the checked class applied. 
5. Close devtools. Look at the element selected in step 2.
6. The element appears with the styling shown in the screenshot above, and the other options appear as expected as well.
7. Hover over the options on the page.
8. The hover color on the unselected options is a grayish color, and the color on the selected option does not change on hover.
9. Refresh the page.
10. Using the keyboard, tab into the options list and change selections by pressing the keyboard's up and down arrows.
11. The styling appears as expected based on the screenshot above.
12. Repeat and validate step 8.
13. Click the "next" button.
14. The next button behaves as expected.
15. On another page with options, click on the "next" button without making a selection.
16. The error message appears as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
7. Find the commit of this pull request.
17. Build and deploy the changes.
18. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
